modules/apps/hooks/hooks.go

```diff
@@ -0,0 +1,142 @@
+package ibc_hooks
```

Comment:
Would it make sense to move this file inside the `types` folder?

Reply:
I think that would create an import cycle, as ibc_module.go depends on the hooks and the hooks on the middleware struct defined in that file